sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -187,7 +187,7 @@ $(document).on('click', '.postCourseNameChange', function (e) {
   e.preventDefault();
   var thisCourse = $(this).attr('data-courseid');
   var newCourseName = $(this).closest('.courseTitleTextContainer').find('input.courseTitleText').val();
-  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;;
+  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;
```

Comment:
I wouldn't work out a way to pass in properties to the html if this is the
only property.  If there are more than one value that should be
configurable then  the problem would be worth revisiting.

On Wed, Mar 4, 2015 at 8:05 AM, Pushyami Gundala <notifications@github.com>
wrote:

> In sectionsUtilityTool/src/main/webapp/assets/js/utils.js
> <https://github.com/tl-its-umich-edu/canvas/pull/17#discussion_r25772559>:
>
> > @@ -187,7 +187,7 @@ $(document).on('click', '.postCourseNameChange', function (e) {
> >    e.preventDefault();
> >    var thisCourse = $(this).attr('data-courseid');
> >    var newCourseName = $(this).closest('.courseTitleTextContainer').find('input.courseTitleText').val();
> > -  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;;
> > +  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;
>
> It's Secure since we have authorization check on Java script files and for
> xhr request we check if the request is from valid user then we process the
> call to canvas.
>
> We don't have a mechanism right now where can send the data from backend
> to UI since we have html page and not JSP page. I can only see one
> possibility is making ajax call requesting Prefix info.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/canvas/pull/17/files#r25772559>.
>